tests/src/model/pages/container-details-page.ts

```diff
@@ -79,17 +79,9 @@ export class ContainerDetailsPage extends BasePage {
     }
   }
 
-  async deleteContainer(timeout: number): Promise<ContainersPage> {
+  async deleteContainer(): Promise<ContainersPage> {
     const deleteButton = this.page.getByRole('button').and(this.page.getByLabel('Delete Container'));
     await deleteButton.click();
-    await waitWhile(
-      async () => await this.heading.isVisible(),
-      timeout,
-      1000,
-      true,
-      `Container was not deleted in ${timeout / 1000}s`,
-    );
-    // after delete is successful we expect to see containers page
     return new ContainersPage(this.page);
```

Comment:
Won't this call fail unless the expected page is present in DOM?

Reply:
Not anymore than it was failing previously, validating navigation should be done outside of functional methods.